pyproject.toml

```diff
@@ -15,12 +15,42 @@ testpaths = [
 name = "greynirseq"
 version = "0.1"
 description = "Natural language processing for Icelandic"
-authors = []
+authors = ["Miðeind ehf <tauganet@mideind.is>"]
```

Comment:
Við erum með setup.py og pyproject.toml sem innihalda mismunandi upplýsingar um pakkann.
pyproject skilgreinir package dependencies en setup.py skilgreinir hvernig á að byggja cython dependencies.

Virka þessar tvær skrár saman?

Reply:
Við getum reynt: https://github.com/mdgoldberg/poetry-cython-example

Reply:
Let's take that as a part of the constituency parsing